chain/jsonrpc/src/lib.rs

```diff
@@ -212,6 +212,24 @@ fn process_query_response(
                 "block_height": block_height,
                 "block_hash": block_hash,
             })),
+            near_jsonrpc_primitives::types::query::RpcQueryError::UnknownBlock {
+                ref block_reference,
+            } => match block_reference {
+                near_primitives::types::BlockReference::BlockId(block_id) => Err(RpcError::new(
+                    -32_000,
```

Comment:
Can we make this status code maybe a 
```
const RPC_ERROR_CODE:i32 = -32_200; 
```
with the name a bit more descriptive as I don't really know what does the number represent? 😄

Reply:
@chefsale Well, this function documentation goes the following:

```
/// This function processes response from query method to introduce
/// backward compatible response in case of specific errors
```

This number is mostly useless and means nothing, so we just keep it there for compatibility reasons.

@khorolets you may probably provide a reference to the `RpcError::server_error` method in a comment.